<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LessonWindow</class>
 <widget class="QWidget" name="LessonWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="nextPushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>800</width>
     <height>750</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="quiz">
    <widget class="QLabel" name="infoLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>801</width>
       <height>500</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="aOption">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>550</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bOption">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>550</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cOption">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>650</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dOption">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>650</y>
       <width>400</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="interactive">
    <widget class="QPushButton" name="amazonPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>AMZN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="vfinxPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>VFIAX</string>
     </property>
    </widget>
    <widget class="QPushButton" name="teslaPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>TSLA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="usbPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>USB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="snapPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>CLX</string>
     </property>
    </widget>
    <widget class="QLabel" name="graphLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>571</width>
       <height>391</height>
      </rect>
     </property>
     <property name="text">
      <string>Graph Place Holder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>620</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>quantity:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="investPushButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>610</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Buy</string>
     </property>
    </widget>
    <widget class="QLabel" name="warningLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>441</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>WARNING: There are four different time periods for you to sell stocks. We recommend you to consider each time period to optimize your profits.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="quantitySpinBox">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>610</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="quantityLabel">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>570</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>quantityLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="quantPriceLabel">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>570</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>quantPriceLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="priceLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>priceLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>user's money: </string>
     </property>
    </widget>
    <widget class="QLabel" name="moneyLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>moneyLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="resultLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>431</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>resultLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="stockLabel">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>201</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>stockLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="completeScreen">
    <widget class="QLabel" name="completionLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>531</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>You Completed _______________!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="completetionButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>340</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Return To Menu</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalResultLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>681</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>totalResultLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="infoPages">
    <widget class="QLabel" name="buyInfoLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>800</width>
       <height>571</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
